README.md

```diff
@@ -94,10 +94,20 @@ open http://$(docker-compose port nginx 80)/admin
 ```
 
 Next access the React frontend where `x` below is the id of the configuration entity you just created in the administrative interface.
-```
+```sh
 open http://$(docker-compose port nginx 80)?id=x
 ```
 
+## Loggin
+The engine use logstash to log messages and these can be seen in the docket setup with the following command.
```

Comment:
The engine **uses** logstash to log messages and these can be seen in the **docker** setup with the following command

Reply:
Fixed